specs/phase0/beacon-chain.md

```diff
@@ -1159,7 +1159,7 @@ def initialize_beacon_state_from_eth1(eth1_block_hash: Bytes32,
     state = BeaconState(
         genesis_time=eth1_timestamp + GENESIS_DELAY,
         fork=fork,
-        eth1_data=Eth1Data(block_hash=eth1_block_hash, deposit_count=len(deposits)),
+        eth1_data=Eth1Data(block_hash=eth1_block_hash, deposit_count=uint64(len(deposits))),
```

Comment:
I guess that should be reverted too, since it's phase0 code?

Reply:
It's phase0 so less changes is better, but you're right that it could use stronger typing. And it doesn't change the behavior, nor does it affect readability all that much, and doesn't introduce new constants or functions. So this seems fine.